packages/overlay/test/restore-focus.test.js

```diff
@@ -1,9 +1,9 @@
 import { expect } from '@esm-bundle/chai';
-import { fixtureSync } from '@vaadin/testing-helpers';
+import { aTimeout, fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import '../src/vaadin-overlay.js';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
-import { close, open } from './helpers.js';
```

Comment:
The tests now open and close the overlay on their own. Although I could technically still use `open()`, I decided against it as I thought it would be more consistent that way. Let me know if you think we should revert that part.

Reply:
Good point, let me remove them.

Reply:
Done.